stable/sonatype-nexus/templates/proxy-svc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "nexus.fullname" . }}
+  labels:
+{{ include "nexus.labels" . | indent 4 }}
+spec:
+  ports:
+    - name: {{ .Values.nexusProxy.name }}
+      port: {{ .Values.nexus.externalPort }}
+      protocol: TCP
+      targetPort: {{ .Values.nexus.externalPort }}
```

Comment:
Should certainly not be the external port. I'd suggest you hard-code container ports and reference them by name. Changing them would break things unless you reconfigure Nexus itself with the changed ports.

Reply:
I think there was some poor naming of the ports (fixed in latest commit). But I think a great benefit of such options in Helm is that it's easy to use a different image that uses a different port. Hopefully the setup is a little more clear on usage.